torch/_inductor/ir.py

```diff
@@ -2478,6 +2478,12 @@ def is_contiguous_strides_for_shape(stride, shape):
     )
 
 
+def is_contiguous_or_transposed(sizes, strides):
```

Comment:
Transposition can happen in any pair of dims, not necessarily the last two dims. Also you are checking `stride >= size` not `stride == size`. The function name doesn't seem to match the implementation here. Perhaps, it is clearer we just inline the implementation in the `tuned_bmm` code without factoring out a util function here.

Reply:
Thanks and modified!